sio/CMakeLists.txt

```
@@ -38,6 +38,7 @@ TARGET_INCLUDE_DIRECTORIES( sio PUBLIC
 TARGET_INCLUDE_DIRECTORIES( sio SYSTEM PUBLIC ${ZLIB_INCLUDE_DIR} )
 TARGET_LINK_LIBRARIES( sio ${ZLIB_LIBRARIES} )
```

```suggestion
TARGET_LINK_LIBRARIES( sio PUBLIC ZLIB::ZLIB )
```
The ZLIB targets are available since cmake 3.1
https://cmake.org/cmake/help/latest/module/FindZLIB.html